Add LYCHEE_UPLOADS_URL to secure-headers.php

hxrsmurf opened the pull request:
Allow one to use the PHP environment variable ```LYCHEE_UPLOADS_URL``` to source images. https://github.com/LycheeOrg/Lychee/issues/533

hxrsmurf committed:
Add LYCHEE_UPLOADS_URL to secure-headers.php

https://github.com/LycheeOrg/Lychee/issues/533

ildyria reviewed (changes requested):
can you please add as a default value `'https://lycheeorg.github.io/'` as stated here: #533 

I will happily merge it afterwards. :+1:

hxrsmurf committed:
Add LYCHEE_UPLOADS_URL to secure-headers.php

Add support for PHP ENV ```LYCHEE_UPLOADS_URL``` with lycheeorg.github.io as default

ildyria reviewed (approved):
LGTM.

hxrsmurf commented:
Thank you very much. I think I did that pull correctly.
